src/components/warning-component.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { StyleSheet, Text, View, Image } from 'react-native';
+import colors from '@styles/colors';
+
+enum warningType {
+    Thunderstorm,
+    Strong_wind,
+    Snowstorm,
+    Rain,
+    Fog,
+    Little_wind,
+    hail,
+}
+
+export function Warning(War: warningType) {
+    switch (War) {
+        case warningType.Thunderstorm:
+            return 'Возможна гроза';
+            break;
+        case warningType.Strong_wind:
+            return 'Возможно сильный ветер';
+            break;
+        case warningType.Snowstorm:
+            return 'Возможна метель';
+            break;
+        case warningType.Rain:
+            return 'Возможно дождь';
+            break;
+        case warningType.Fog:
+            return 'Возможно туман';
+            break;
+        case warningType.Little_wind:
+            return 'Возможно слабый ветер';
+            break;
+        case warningType.hail:
+            return 'Возможно град';
+            break;
+        default:
+            return 'Предупреждений нет';
+    }
+}
+
+export default function Shape() {
+    return (
+        <View>
+            <View style={styles.appearance}>
+                <Image source={require('@assets/icon/flash_24.png')} style={styles.icon} />
+                <Text style={styles.text}>{Warning(warningType.Thunderstorm)}</Text>
+            </View>
+        </View>
+    );
+}
+
+const styles = StyleSheet.create({
+    icon: {
+        width: 24,
+        height: 24,
+        marginHorizontal: 14,
+    },
+    appearance: {
+        justifyContent: 'space-around',
+        flexDirection: 'row',
+        alignItems: 'center',
+        backgroundColor: colors.LIGHT_ORANGE,
+        width: 327,
+        height: 52,
```

Comment:
As discussed you MUSTN'T use absolute values for something like this, because it would be different on different device screens, so for width use %, however for height, margins and paddings you can use absolute values. We need everything to have same offsets, so it is better to create file constants in styles and describe some frequently used offsets, for example in this component we will need same offsets from the edge, so we can create horizontal margin constant. Then add this horizontal margin for wrapper.
![image](https://user-images.githubusercontent.com/52620425/223999279-66ce1614-1a38-455b-ab24-989f2d0ec790.png)
Also it is better to use name convention again
![image](https://user-images.githubusercontent.com/52620425/224000053-3a1fef3f-4cd5-4072-ac51-b048a3a8296d.png)